src/action.js

```diff
@@ -35,11 +35,26 @@ module.exports = async function run({
     APPROVE_ONLY,
     USE_GITHUB_AUTO_MERGE,
     TARGET,
+    TARGET_DEV,
+    TARGET_PROD,
     PR_NUMBER,
     SKIP_COMMIT_VERIFICATION,
     SKIP_VERIFICATION,
   } = getInputs(inputs)
 
+  let target = TARGET
+  if (
+    dependabotMetadata.dependencyType === 'direct:development' &&
+    TARGET_DEV
+  ) {
+    target = TARGET_DEV
+  } else if (
+    dependabotMetadata.dependencyType === 'direct:production' &&
+    TARGET_PROD
+  ) {
+    target = TARGET_PROD
+  }
```

Comment:
let's create a function for this please, it clutters the logic of this function with implementation details that don't fit here, besides using mutable state which is an ugly pattern